packages/@react-spectrum/utils/test/Slots.test.js

```diff
@@ -190,4 +191,97 @@ describe('Slots', function () {
     expect(getByRole('presentation')).toHaveAttribute('aria-controls', id);
   });
 
+  // tests are wrapped in strict mode so renders are double what you'd typically expect
+  const STRICT_MODE_RENDER_MULTIPLIER = 2;
```

Comment:
I'm not aware of any convention for testing something like this. It's seemingly unavoidable that these tests rely on strict mode implementation details.

My intention here is to keep behavior consistent between running all tests and running in isolation. All tests are wrapped with StrictMode globally so I wrapped it again locally for isolated test runs which I think is just a no-op when all tests are run